test/js/web/websocket/websocket-client-short-read.test.ts

```diff
@@ -0,0 +1,101 @@
+import { describe, test, expect } from "bun:test";
+
+const hostname = process.env.HOST || "127.0.0.1";
+const port = parseInt(process.env.PORT || "0");
+
+describe("WebSocket", () => {
```

Comment:
looks like this test is timing out on linux

Reply:
I can't seem to see the CI debug logs. Are you testing it locally? If so, could you share the logs?